### DISCUSSION TITLE
Transform AWS::Serverless-2016-10-31 failed with: Invalid Serverless Application Specification document

### DISCUSSION THREAD

**Discussion by ipeakermit** [mozilla/hubs#5448] [Hubs Cloud Questions]
When I try to stand up Hubs Enterprise stack I get the following error; does anyone have any suggestions on the cause?

Transform AWS::Serverless-2016-10-31 failed with: Invalid Serverless Application Specification document. Number of errors found: 5. Resource with id [SESDomainApplication] is invalid. Property 'ApplicationId' cannot be resolved. Only FindInMap and Ref intrinsic functions are supported. Resource with id [KeymasterApplication] is invalid. Property 'ApplicationId' cannot be resolved. Only FindInMap and Ref intrinsic functions are supported. Resource with id [NearsparkApplication] is invalid. Property 'ApplicationId' cannot be resolved. Only FindInMap and Ref intrinsic functions are supported. Resource with id [SpeelycaptorApplication] is invalid. Property 'ApplicationId' cannot be resolved. Only FindInMap and Ref intrinsic functions are supported. Resource with id [PhotomnemonicApplication] is invalid. Property 'ApplicationId' cannot be resolved. Only FindInMap and Ref intrinsic functions are supported.. Rollback requested by user.

**Comment by pattersonbl2** [comment]
What cloud formation are you using?   This seems like the application id for the actual resource is wrong or missing.

**Comment by ipeakermit** [comment]
The Template URL appears to be: https://s3.amazonaws.com/awsmp-fulfillment-cf-templates-prod/94ec4ef1-285d-4e63-b2eb-0b65ea93b942.4c0a54df-64aa-4a31-ab81-20b1c27f66eb.template .

Let me know if you need more detail---I'm a bit familiar with hubs, having tried and failed to get a self-hosted instance up and running but not terribly familiar with AWS and CloudFormation.

**Reply by pattersonbl2** [reply]
What region are you trying to run this in ?  everything looks correct in the template.  The error messages are normally related to the cloudformation not being able to match the arn provided in the template.  This could be if you are trying to deploy in a region that the template doesn't support.

**Comment by ipeakermit** [comment]
RMIT Classification: Trusted

Thanks, sounds plausible.

I'm trying to run in AsiaPacific/Sydney

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by pattersonbl2** [reply]
Yeah, We don't have hubs in that region for hubs currently which would cause issues

**Reply by ipeakermit** [reply]
Is there something we can work towards to address this? Are there specific differences which make portability between regions complex? Is it possible we could create/modify the template?
